elements/rocky-container-stackhpc/README.rst

```diff
@@ -2,7 +2,15 @@
 rocky-container-stackhpc
 ========================
 Custom containerfiles for usage with ``rocky-container`` builds.
+
 Usage:
 Set ``DIB_CONTAINERFILE_DOCKERFILE`` environment variable to custom
 Containerfile path provided by this role, e.g.:
 DIB_CONTAINERFILE_DOCKERFILE: "/opt/kayobe/src/stackhpc-image-elements/elements/rocky-container-stackhpc/containerfiles/9-stackhpc"
+
+Set ``ROCKY_USE_MIRRORS=true`` to disable upstream Rocky repository mirrors, and
```

Comment:
I think `ROCKY_USE_MIRRORS` could be slightly confusing. Might make more sense as `ROCKY_USE_CUSTOM_MIRRORS`?

Reply:
Agreed, updated to use `ROCKY_USE_CUSTOM_DNF_MIRRORS` and `ROCKY_CUSTOM_DNF_MIRROR_URLS`.